doc/guides/backporting-to-release-lines.md

```diff
@@ -0,0 +1,81 @@
+# How to backport a Pull Request to a Release Line
+
+## Staging branches
+
+Each release line has a staging branch that the releaser will use a scratch pad
+while preparing a release. The branch name is formatted as follows:
+`vn.x-staging` where `n` is the major release number.
+
+### Active staging branches
+
+| Release Line | Staging Branch |
+| ------------ | -------------- |
+| `v7.x`       | `v7.x-staging` |
+| `v6.x`       | `v6.x-staging` |
+| `v4.x`       | `v4.x-staging` |
+
+## What needs to be backported?
+
+When a release is being prepared, the releaser attempts to cherry-pick a
+certain set of commits from the master branch to the release staging branch.
+If a cherry-pick does not landing cleanly on the staging branch, the releaser
```

Comment:
Yes, when landing a commit for pushing to master, we use `git am`. For backporting, the goal is to get the commit unmodified, so we use `git cherry-pick <sha>`.